src/component/feature/feature-list-item-component.jsx

```diff
@@ -42,12 +43,26 @@ const Feature = ({
                 <Progress strokeWidth={15} percentage={percent} isFallback={isStale} />
             </span>
             <span className={styles.listItemToggle}>
-                <Switch
-                    disabled={!updateable || toggleFeature === undefined}
-                    title={`Toggle ${name}`}
-                    key="left-actions"
-                    onChange={() => toggleFeature(name)}
-                    checked={enabled}
+                <PermissionComponent
+                    permission={UPDATE_FEATURE}
+                    component={
+                        <Switch
+                            disabled={toggleFeature === undefined}
+                            title={`Toggle ${name}`}
+                            key="left-actions"
+                            onChange={() => toggleFeature(name)}
+                            checked={enabled}
+                        />
+                    }
+                    otherwise={
+                        <Switch
+                            disabled
+                            title={`Toggle ${name}`}
+                            key="left-actions"
+                            onChange={() => toggleFeature(name)}
+                            checked={enabled}
+                        />
+                    }
```

Comment:
Having a `Permission` component makes it very explicit which is :+1: 

But it feels a bit verbose when you just want to disable a button. Would it make sense to also provide a `hasPermission` function which can be used in these scenarios? 

side:
If we keep the `otherwise` component: Does it not make sense to remove the `onChange` function and add use-feedback on why the component is disabled?